creativecloud/scripts/scripts.js

```diff
@@ -124,6 +124,7 @@ const CONFIG = {
   locales,
   geoRouting: 'on',
   prodDomains: ['www.adobe.com'],
+  queryIndexCardPath: '/cc-shared/assets/query-index-cards',
```

Comment:
This is not going to work for other countries/languages once rolled out. The query index for cards in its current state is only setup for `us` only at the moment, see https://github.com/adobecom/cc/blob/main/helix-query.yaml#L119. This needs to be expanded at some point.